### PR TITLE
Remove sleep, and switch to actual assertion within admin order edit test.

### PR DESCRIPTION
**Pull request by vinniefranco**
#### Why?
The admin order edit spec was randomly failing on Travis... 

#### What's changed?
This PR removes sleep in effort to deal with random failures, and changes the spec to assert that stock is in fact updated correctly.

### CODE REVIEW

File: spec/features/admin/orders_spec.rb [hunk 1]
```diff
@@ -12,18 +12,38 @@
...
 
   it "allows admin to edit product bundle" do
+    change_orders_item_quantity_to 2
+
+    visit spree.edit_admin_order_path(order)
+
+    verify_3_parts_have_their_quantity_set_to 2
+  end
+
+  def change_orders_item_quantity_to(n)
```
Comment (elle) [inline spec/features/admin/orders_spec.rb]: This method should be moved to the bottom and be `private`

File: spec/features/admin/orders_spec.rb [hunk 2]
```diff
@@ -12,18 +12,38 @@
...
     visit spree.edit_admin_order_path(order)
 
     within("table.product-bundles") do
       find(".edit-line-item").click
-      fill_in "quantity", :with => "2"
+      fill_in "quantity", with: n.to_s
```
Comment (elle) [inline spec/features/admin/orders_spec.rb]: Does it need to be a string? or can it just be a number?

File: spec/features/admin/orders_spec.rb (hunk 1 above)
Comment (elle) [inline spec/features/admin/orders_spec.rb]: Actually both those methods should be private, so no need to move anything

File: spec/features/admin/orders_spec.rb (hunk 2 above)
Comment (vinniefranco) [inline spec/features/admin/orders_spec.rb]: Not sure, was just keeping what was there before. I'll try it out!

File: spec/features/admin/orders_spec.rb (hunk 1 above)
Comment (elle) [inline spec/features/admin/orders_spec.rb]: Second actually, since there is only one spec that is using these helper methods, do we really need them??

**Comment by elle** [comment]
:rocket:

**Comment by joneslee85** [comment]
:100:

**Pull request merged**